lib/src/zenohUTransport.cpp

```diff
@@ -33,6 +33,7 @@
 #include <zenoh.h>
 #include <up-core-api/uattributes.pb.h>
 
+
```

Comment:
Minor: There are lots of whitespace changes in this PR that aren't actually relevant to the change. Would it be very difficult to undo those to minimize the number of irrelevant changes?

Reply:
I'm trying to make changes to limit the whitespace differences, but it seems like a bit of a burden, given that I don't understand our whitespace policy, and we don't have a code formatter. Given that the code is fundamentally broken, and will need massive changes once we get past this, I would suggest that we put a priority on choosing a code formatter, having, and reformatting PR, and then insist that all PRs are code formatted with the agreed-upon formatter with every PR.

Reply:
The ask is simply not to change whitespace in sections of the code where you are not making actual code changes. No formatter should be needed to not change things. For example, the whitespace changes pointed out here are adding / removing blank lines between code lines.

Reply:
Looking at the latest files, I think we can call this resolved. Thanks for cleaning that up.